src/DynamoCoreWpf/Utilities/CrashReportTool.cs

```diff
@@ -194,7 +210,13 @@ internal static void OnCrashReportWindow(CrashReportArgs args)
                 var cerDir = Directory.CreateDirectory(Path.Combine(Path.GetTempPath(), "DynamoCER_Report_" +
                     DateTime.Now.ToUniversalTime().ToString("yyyy-MM-dd-HH-mm-ss")));
 
-                using (Scheduler.Disposable.Create(() => { cerDir.Delete(); }) )
+                using (Scheduler.Disposable.Create(() => {
+                    // Cleanup
+                    foreach (FileInfo file in cerDir.EnumerateFiles())
```

Comment:
these have the potential to throw, maybe wrap in a try catch?

Reply:
It is...but it might be useful to try catch the cleanup separately...since the outer catch will cause the backup (old) crash prompt to be displayed